pkgs/build-support/node/fetch-yarn-deps/tests/file.lock

```diff
@@ -0,0 +1,9 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@org/somepack@file:vendor/orgpacks/somepack/assets":
```

Comment:
Please help me to understand the example. In the log of this test, I see only one line `ignoring lockfile entry "otherpack" which points at path "vendor/otherpack"`. Why does the `somepack` not enter the `ignoring lockfile ...` code path?

Reply:
You're right. There was another bug in https://github.com/NixOS/nixpkgs/blob/11f4e814f624face2e8c775a8bd892be6b8b811f/pkgs/build-support/node/fetch-yarn-deps/index.js#L140 which accesses `pkg.resolved`, which will fail for `file:...` entries. I took a closer look and simplified that function...